src/components/Stepper/Stepper.tsx

```diff
@@ -1,14 +1,18 @@
 import { type ReactElement } from "react";
 
-const Stepper = () => {
+type StepperProps = {
+  step: number;
+};
+
+const Stepper = ({ step }: StepperProps) => {
   const renderSteps = (): ReactElement[] => {
     const steps: number = 4;
```

Comment:
: number this is not needed. TS will infer this type automatically https://www.typescriptlang.org/docs/handbook/type-inference.html